[2021-06-30 08:49:26,365] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: open_aq.store_data 2021-06-29T17:09:19.209771+00:00 [queued]>
[2021-06-30 08:49:26,397] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: open_aq.store_data 2021-06-29T17:09:19.209771+00:00 [queued]>
[2021-06-30 08:49:26,398] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-06-30 08:49:26,399] {taskinstance.py:1068} INFO - Starting attempt 13 of 13
[2021-06-30 08:49:26,401] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-06-30 08:49:26,426] {taskinstance.py:1087} INFO - Executing <Task(PythonOperator): store_data> on 2021-06-29T17:09:19.209771+00:00
[2021-06-30 08:49:26,433] {standard_task_runner.py:52} INFO - Started process 7112 to run task
[2021-06-30 08:49:26,451] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'open_aq', 'store_data', '2021-06-29T17:09:19.209771+00:00', '--job-id', '774', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/open_aq.py', '--cfg-path', '/tmp/tmpc_yduexg', '--error-file', '/tmp/tmpcm8tyb5m']
[2021-06-30 08:49:26,455] {standard_task_runner.py:77} INFO - Job 774: Subtask store_data
[2021-06-30 08:49:26,574] {logging_mixin.py:104} INFO - Running <TaskInstance: open_aq.store_data 2021-06-29T17:09:19.209771+00:00 [running]> on host airflowvm
[2021-06-30 08:49:26,691] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=open_aq
AIRFLOW_CTX_TASK_ID=store_data
AIRFLOW_CTX_EXECUTION_DATE=2021-06-29T17:09:19.209771+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-06-29T17:09:19.209771+00:00
[2021-06-30 08:49:26,715] {base.py:69} INFO - Using connection to: id: ***_default. Host: localhost, Port: 5432, Schema: , Login: ***, Password: ***, extra: {'cursor': 'realdictcursor'}
[2021-06-30 08:49:26,727] {python.py:151} INFO - Done. Returned value was: None
[2021-06-30 08:49:26,764] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=open_aq, task_id=store_data, execution_date=20210629T170919, start_date=20210630T084926, end_date=20210630T084926
[2021-06-30 08:49:26,811] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-06-30 08:49:26,825] {local_task_job.py:151} INFO - Task exited with return code 0
